### DISCUSSION TITLE
Unable to get code coverage for a WordPress and php web application having cypress E2E tests

### DISCUSSION THREAD

**Discussion by cypress28** [cypress-io/cypress#27901] [Questions and Help]
Hello,

I have a web application which has its source code in php and uses wordpress as a third party tool to render on the web browser. I have couple of cypress end to end tests which primarily test if the web applications looks as expected. I integrated @cypress/code-coverage plugin but there is no code coverage generated apart form cypress.config.js file (100%)
I am not sure if code coverage will be generated for a php-wordpress application or something is wrong with installation and configuration of the plugin.

Can someone please help?

Steps 
1. Installed the plugin
2. Import and register the plugin in index.js
3. Modified cypress.config.js configuration file to include the coverage settings
4. Install Babel and Istanbul/NYC
5. Created .babelrc file in your project's root directory and include the babel-plugin-istanbul plugin for code coverage instrumentation.
6.  Add configuration in your package.json for the report to generate in .nyc_output directory
7.  Ran tests with command  npx nyc cypress run

**Comment by ThomasMeschke** [comment]
Hi. 
I was on the lookout for the same and as far as I got the documentation, cypress can only measure coverage across Javascript Code and will never provide coverage across PHP code. 
Which makes absolute sense when you think about it, because coverage requires the code to be instrumented to take measurements. And I don't think cypress has the proper skills to instrument PHP code.
